generator/geo_objects/geo_objects.hpp

```diff
@@ -1,25 +1,135 @@
 #pragma once
 
+#include "generator/key_value_storage.hpp"
+#include "generator/regions/region_info_getter.hpp"
+
+#include "geometry/meter.hpp"
+#include "geometry/point2d.hpp"
+
+#include "base/logging.hpp"
+#include "base/scope_guard.hpp"
+#include "base/timer.hpp"
+
+#include "platform/platform.hpp"
+
 #include <string>
 
 namespace generator
 {
 namespace geo_objects
 {
-// This function generates key-value pairs for geo objects.
-// First, we try to generate key-value pairs only for houses, since we cannot say anything about poi.
-// In this step, we need key-value pairs for the regions and the index for the regions.
-// Then we build an index for houses. And then we finish building key-value pairs for poi using
-// this index for houses.
-// |allowAddresslessForCountries| specifies countries for which addressless buldings are constructed
-// in index and key-value files. Countries are specified by osm's default local name (or part of name)
-// separated by commas. Default value is '*' (for all countries).
-bool GenerateGeoObjects(std::string const & pathInRegionsIndex,
-                        std::string const & pathInRegionsKv,
-                        std::string const & pathInGeoObjectsTmpMwm,
-                        std::string const & pathOutIdsWithoutAddress,
-                        std::string const & pathOutGeoObjectsKv,
-                        std::string const & allowAddresslessForCountries,
-                        bool verbose, size_t threadsCount);
+namespace
```

Comment:
Can we avoid using anonymous namespace in a header file here?

Reply:
OK, thanks! Seems to be cleaner header file.